sycl/doc/developer/DockerBKMs.md

```diff
@@ -78,7 +74,8 @@ development containers:
    NVidia/AMD and can be used for building DPC++
    compiler from source with all backends enabled or for end-to-end testing
    with HIP/CUDA on machines with corresponding GPUs available.  
-
+ - `devops/containers/nightly`: contains the latest successfully
+   built nightly build of DPC++ compiler.
```

Comment:
```suggestion
   built nightly build of DPC++ compiler, along with `latest` Intel drivers and NVidia/AMD development kits (same as `alldeps`).
```

Reply:
I've done that on purpose. I'm planning to fuse tags for the base images as well, so that we'll only have 2204/2404/nightly images and once done that clarification will become irrelevant and even confusing.

Reply:
Sure, seems reasonable to me.